src/gui_tool/gui/sp_gui.py

```diff
@@ -269,25 +269,25 @@ def toggle_status(self):
 class SplitManager(object):
     # Color in BGR
     SPLIT_DISPLAY = {
-        "fontFace": cv2.FONT_HERSHEY_SIMPLEX,
+        "fontFace": cv2.FONT_HERSHEY_DUPLEX,
         "fontScale": 1,
         "thickness": 2,
         "color": (153, 51, 51)
     }
     DELETED_DISPLAY = {
-        "fontFace": cv2.FONT_HERSHEY_SIMPLEX,
+        "fontFace": cv2.FONT_HERSHEY_DUPLEX,
         "fontScale": 1,
         "thickness": 2,
         "color": (10, 10, 204)
     }
     ACTIVE_DISPLAY = {
-        "fontFace": cv2.FONT_HERSHEY_SIMPLEX,
+        "fontFace": cv2.FONT_HERSHEY_DUPLEX,
         "fontScale": 1,
         "thickness": 2,
         "color": (51, 204, 51)
     }
     ENUM_DISPLAY = {
-        "fontFace": cv2.FONT_HERSHEY_SIMPLEX,
+        "fontFace": cv2.FONT_HERSHEY_DUPLEX,
```

Comment:
whats the difference here? I find the font is kinda blurry at least on linux
![image](https://user-images.githubusercontent.com/20507588/108616315-65a04880-73da-11eb-8000-d71f786f6e8b.png)

Reply:
When resizing the window, the logger text will get jagged/aliased, making it harder to read since it's so thin. The only difference is that this font is thicker, and imo, easier to read.